src/plugins/intel_npu/src/plugin/npuw/compiled_model.cpp

```diff
@@ -84,10 +86,30 @@ ov::npuw::DeviceProperties get_properties_per_device(const std::shared_ptr<const
 }  // namespace npuw
 }  // namespace ov
 
+std::shared_ptr<ov::npuw::ICompiledModel>
+ov::npuw::CompiledModelFactory::create(const std::shared_ptr<ov::Model>& model,
+                                       const std::shared_ptr<const ov::IPlugin>& plugin,
+                                       const ov::AnyMap& properties) {
+    LOG_DEBUG("CompiledModelFactory::create");
```

Comment:
Also, to avoid discrepancies, `__PRETTY_FUNCTION__` may be the macro you'd want to use here

Reply:
Thank you!